Feature/hecuba measurements api

jasebell opened the pull request.

Commit:
Added hecuba api call via http

Commit:
WIP: Fix outgoing messages to API.

Commit:
Check payload counts before attempting to send

tcoupland commented on src/kixi/hecuba/onyx/jobs/measurements_connector.clj:
```diff
@@ -4,10 +4,36 @@
+    (push-payload-to-hecuba measurements entity-id device-id)
+    (push-payload-to-hecuba degree-day entity-id device-id))
+
   (println (str "k.h.o.j.hf - data - " fn-data)))
```
Still println'ing? Feel like this should be a debug log call.

tcoupland replied:
Also wondering if these two pushes should be separate tasks. Get all parallel on it.

jasebell replied:
Nice catch on the println, missed that one and also noticed that log call is an old one which should be a timbre/info call instead.

tcoupland commented:
Made a few comments, but not sure we really need to dial this up that far. All looks nice a tidy in the main!

jasebell commented on src/kixi/hecuba/onyx/jobs/measurements_connector.clj:
```diff
@@ -2,14 +2,49 @@
+      (push-payload-to-hecuba degree-day entity-id device-id)
+      (timbre/info (format "Writing degree day reading to device %s on entity %s" device-id entity-id))))
+  {:done true})
 
```
Just to confirm Tom, the println/timbre changes were already done. So I'm happy to merge.

Commit:
Create docker deployment of application.

- Create dockerfile.
- Include Onyx run scripts.
- Amend config ports to use 40201.
- Expose docker ports 40201 and not class with other Onyx job
  port sharing.
